src/main/java/edu/ucsb/cs156/courses/controllers/CourseOverTimeBuildingController.java

```diff
@@ -53,6 +58,54 @@ public ResponseEntity<String> search(
         convertedSectionCollection.findByQuarterRangeAndBuildingCode(
             startQtr, endQtr, buildingCode);
     String body = mapper.writeValueAsString(courseResults);
+
+    return ResponseEntity.ok().body(body);
+  }
+
+  // new
```

Comment:
Comments like these are obsolete as soon as they are merged.  It's better to let the code speak for itself.  Can we remove this?
```suggestion
```